docs/README.md

```diff
@@ -0,0 +1,86 @@
+# Documentation
+
+<small>[MathType Web Integrations](../README.md) → Documentation</small>
+
+Mono-repository for the [MathType](http://www.wiris.com/en/mathtype) Web plugins and their dependencies.
+
+## Table of contents
+
+- [Requirements](#requirements)
+- [Structure](#structure)
+- [First steps](#first-steps)
+  - [Cloning the repository](#cloning-the-repository)
+  - [Bootstrapping](#bootstrapping)
+- [Next steps](#next-steps)
+- [Technical Support](#technical-support)
```

Comment:
Technical suport, privacy policity and licencse are not present on the document and the links don't work